src/components/zen-avatar-details/zen-avatar-details.tsx

```diff
@@ -34,6 +34,9 @@ export class ZenAvatarDetails {
   /** Set the style variant  */
   @Prop() readonly variant: AvatarDetailVariant = 'detailed';
 
+  /** Set image URL */
```

Comment:
NIT: I don't like this commanding tone ;)
```suggestion
  /** Avatar image URL */
```